runtime/gc_base/modronapi.cpp

```diff
@@ -1116,7 +1116,9 @@ j9gc_prepare_for_checkpoint(J9VMThread *vmThread)
 		verboseGCManager->prepareForCheckpoint(env);
 	}
 
+	releaseVMAccess(vmThread);
```

Comment:
Would you please add a comment why VM Access should be released? Something like: "threads being terminated may trigger a hook that may acquire VM access via JVMTI callback"

Reply:
Added comment with this explanation.

Reply:
may acquire exclusive VM access

Reply:
Updated wording in https://github.com/eclipse-openj9/openj9/commit/6ab4114e3a49f0db2f43f56f6945de2bd6ad8fd3